File: Data Structures & Algorithms/find-duplicate-integer/submission-5.java
class Solution {
    public int findDuplicate(int[] nums) {
        int ans=-1;
        Set<Integer> set = new HashSet<>();
        for(int num: nums){
            if(set.contains(num)){
                ans = num;
            }
            set.add(num);
        }
        return ans;
    }
}

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-5.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        sett = set()
        ans=0

        for ele in nums:
            sett.add(ele)

        for ele in nums:
            if ele-1 not in sett:
                num=ele
                count=0
                while num in sett:
                    num+=1
                    count+=1
                ans=max(ans,count)
        return ans


        